Penumbra/UI/ModsTab/ModPanelTagsFeatureTab.cs

```diff
@@ -0,0 +1,211 @@
+﻿using System;
+using ImGuiNET;
+using OtterGui.Raii;
+using OtterGui;
+using OtterGui.Widgets;
+using Penumbra.Mods.Manager;
+using System.Numerics;
+using System.Collections.Generic;
+using Dalamud.Logging;
+using System.Linq;
+using Penumbra.UI.Classes;
+
+namespace Penumbra.UI.ModsTab;
+
+public class ModPanelTagsFeatureTab : ITab
+{
+    
+    private readonly ModFileSystemSelector _selector;
+    private readonly TutorialService _tutorial;
+    private readonly ModManager _modManager;
+    private readonly TagButtons _localTags = new();
+
+    //Contains the tags
+    private List<List<string>> _tagsList = new();
+    //private Dictionary<string, bool> _tagDict = new();
+    //private List<string> _selectedTags = new();
+
+    public ModPanelTagsFeatureTab(ModFileSystemSelector selector, TutorialService tutorial, ModManager modManager)
+    {
+        _selector = selector;
+        _tutorial = tutorial;
+        _modManager = modManager;
+
+        AddTagsToList();
+    }
+
+    public ReadOnlySpan<byte> Label
```

Comment:
We'd have to figure out where to actually put this. Possibly remove tags from the description tab entirely and just make a `Tags` tab with this functionality. (Would definitely need to display Mod Tags too, though).